Index: configure.ac
===================================================================
--- configure.ac	(révision 3465)
+++ configure.ac	(copie de travail)
@@ -914,6 +914,8 @@
 win32/nsis/lasso-full.nsi
 win32/nsis/lasso-lite.nsi
 win32/nsis/python.nsi
+bindings/Makefile
+bindings/php5/Makefile
 ])
 
 languages_available=`echo $languages_available | sed -e "s/^ //" `
Index: Makefile.am
===================================================================
--- Makefile.am	(révision 3465)
+++ Makefile.am	(copie de travail)
@@ -19,9 +19,10 @@
 
 SUBDIRS		= lasso \
 		  swig \
-		  $(PHP4_SD) $(PHP5_SD) $(PYTHON_SD) $(JAVA_SD) $(PERL_SD) \
+		  $(PYTHON_SD) $(JAVA_SD) $(PERL_SD) \
 		  tests \
 		  win32 \
+		  bindings \
 		  docs
 
 ABS_BUILDDIR	= $(shell pwd)
=== modified file 'Makefile.am'
--- bindings/Makefile.am	2007-11-20 15:18:30 +0000
+++ bindings/Makefile.am	2008-01-02 14:02:46 +0000
@@ -1,4 +1,4 @@
-SUBDIRS = python php5
+SUBDIRS = php5
 
 EXTRA_DIST = bindings.py \
 	lang_python.py \

